services/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM node:9 as builder
+FROM node:10 as builder
```

Comment:
Not sure how this is related, this looks like it's not, and it should have been a separate PR. (?)